modules/jupyternotebook/meta.yml

```diff
@@ -0,0 +1,47 @@
+name: jupyternotebook
+## TODO nf-core: Add a description of the module and list keywords
+description: write your description here
+keywords:
+  - sort
+tools:
+  - jupyternotebook:
+      ## TODO nf-core: Add a description and other details for the software below
+      description: 
+      homepage: 
+      documentation: 
+      tool_dev_url: 
+      doi: ""
+      licence: 
+
+## TODO nf-core: Add a description of all of the variables used as input
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  ## TODO nf-core: Delete / customise this example input
+  - bam:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+
+## TODO nf-core: Add a description of all of the variables used as output
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - version:
+      type: file
+      description: File containing software version
+      pattern: "*.{version.txt}"
+  ## TODO nf-core: Delete / customise this example output
+  - bam:
+      type: file
+      description: Sorted BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+
+authors:
+  - "@grst"
```

Comment:
Please refer to the `rmarkdown` variant of `meta.yml` for now. They are highly redundant, and let's discuss it in a single place first.